### PR TITLE
Fix topics with periods for hive #136

### PR DESCRIPTION
**Pull request by ig-michaelpearce**
Fix issue with topic names containing dots, not being compatible with hive (replace dot with underscore)

### CODE REVIEW

**Comment by ig-michaelpearce** [comment]
can someone signpost me to where we can sign the "Contributor License Agreement"

**Comment by ig-michaelpearce** [comment]
Hi robot, 
[clabot:check]

**Comment by ig-michaelpearce** [comment]
Feed back  for whom ever maintains the CLABOT.

It looks like the CLABOT used to output a more useful message directing people to where to sign the CLA. The above auto reply having no link or details isn't that useful for someone coming to the repo, making any new contributors having to hunt.

I found the link only due to an old automated reply https://github.com/confluentinc/kafka-connect-hdfs/pull/70

File: src/main/java/io/confluent/connect/hdfs/hive/HiveMetaStore.java [hunk 1]
```diff
@@ -16,18 +16,23 @@
 
 package io.confluent.connect.hdfs.hive;
 
+import io.confluent.connect.hdfs.FileUtils;
+import javafx.scene.control.Tab;
```
Comment (cotedm) [inline src/main/java/io/confluent/connect/hdfs/hive/HiveMetaStore.java]: Can you cleanup these imports? I don't see them used anywhere.

**Comment by ig-michaelpearce** [comment]
@cotedm as discussed new PR with review changes due to deleted/re-forked local repo - https://github.com/confluentinc/kafka-connect-hdfs/pull/164

**Comment by cotedm** [comment]
Superceded by #164

**Pull request closed without merging**